gis/grass7/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           wxWidgets 1.0
 PortGroup           github 1.0
 
-github.setup        OSGeo grass 7.8.2
+github.setup        OSGeo grass 7.8.3
 name                grass7
 set main_version    [join [lrange [split ${version} "."] 0 1] ""]
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Changed, missed that one. Thanks!